cli/src/cluster_query.rs

```diff
@@ -1044,6 +1075,41 @@ pub fn process_leader_schedule(
     }))
 }
 
+pub fn process_get_recent_priority_fees(
+    rpc_client: &RpcClient,
+    config: &CliConfig,
+    accounts: &[Pubkey],
+    num_slots: Option<Slot>,
+) -> ProcessResult {
+    let mut ret = String::new();
+    if let Ok(fees) = rpc_client.get_recent_prioritization_fees(accounts) {
+        let mut min = u64::MAX;
+        let mut max = 0;
+        let mut total = 0;
+        if config.verbose {
+            ret.push_str(&format!("{:<11} {}\n", "slot", "prioritization_fee"));
+        }
+        let num_slots = num_slots
+            .unwrap_or(fees.len() as u64)
+            .min(fees.len() as u64) as usize;
+        for fee in fees.iter().skip(fees.len() - num_slots) {
+            min = min.min(fee.prioritization_fee);
+            max = max.max(fee.prioritization_fee);
+            total += fee.prioritization_fee;
+            if config.verbose {
+                ret.push_str(&format!("{:<11} {}\n", fee.slot, fee.prioritization_fee));
+            }
+        }
+        ret.push_str(&format!(
+            "Fees in recent {num_slots} slots: Min: {} Max: {} Average: {:.2}",
+            min,
+            max,
+            total as f32 / num_slots as f32
```

Comment:
sorry for additional changes,`num_slots` could be zero, need to prevent divide by zero.

Also, is there strong reason to print average as f32? Otherwise, I'd think have average in whole `micro-lamports` is easier to reason about.